OpenRA.Mods.Common/Traits/Cloak.cs

```diff
@@ -26,11 +38,9 @@ public class CloakInfo : UpgradableTraitInfo
 		[Desc("Measured in game ticks.")]
 		public readonly int CloakDelay = 30;
 
-		public readonly bool UncloakOnAttack = true;
-		public readonly bool UncloakOnMove = false;
-		public readonly bool UncloakOnUnload = true;
-		public readonly bool UncloakOnInfiltrate = true;
-		public readonly bool UncloakOnDemolish = true;
+		[Desc("Events leading to the actor getting uncloaked.")]
```

Comment:
You need to document the possible values here for our auto-generated https://github.com/OpenRA/OpenRA/wiki/Traits documentation. It is inconvenient. I filed https://github.com/OpenRA/OpenRA/issues/10378.